Makefile

```diff
@@ -791,9 +791,9 @@ setup: venv
 	$(MAKE) -C tools
 	$(PYTHON) tools/decompress_baserom.py $(VERSION)
 	$(PYTHON) tools/extract_baserom.py $(BASEROM_DIR)/baserom-decompressed.z64 $(EXTRACTED_DIR)/baserom -v $(VERSION)
+	$(PYTHON) -m tools.assets.extract $(EXTRACTED_DIR)/baserom $(EXTRACTED_DIR) -v $(VERSION) -j
```

Comment:
2 things:
1. Personally I think it would still be better to have a tools/extract_assets.py even if it just passes things right off onto `tools.assets.extract`. Though looks like you have a `tools/extract_asset.sh` any reason not to use that here?
2. Not a big fan of just `-j`. I personaly prefer being able to have the ability to tell how many threads to use so think the `N_THREADS` should be put back

Reply:
1. I think it's more self-documenting to have the makefile pass all paths to tools rather than tools constructing the paths themselves, it shows better how the bricks assemble. extract_assets.sh is there for convenience to extract assets outside of make setup

2. sure

Reply:
1. I meant more running a script rather than calling tools.assets.extract as a python module (it just seems weird for just this one thing to run as a module vs a script like the others, though that is probably just a me thing). I know we've had this conversation multiple times, though it seems to have always been punted. Reading through discord it does look like you actually need to run it as a module for relative imports, so I guess this should probably just be disregarded.

Reply:
Yeah the tool needs to run as a module for relative (and absolute) imports.